src/tests/runTest.ts

```diff
@@ -18,7 +18,6 @@
 // All tests ran here are ones that require the vscode API
 import * as path from 'path'
 import * as cp from 'child_process'
-import { parseArgs } from 'util'
```

Comment:
This a node 19 thing only. The updated way should work for all nodejs version. Not completely sure how the CI didn't fail because of this, wondering if yarn.lock didn't get fully updated correctly.

Reply:
I like your change.  We don't need a very powerful argument parser here (yet).

Reply:
Yeah I mean if we need something more in depth it would possibly be best to do it ourselves.